src/api/middlewares/validator.js

```diff
@@ -16,23 +16,24 @@ const verifyParams = (req, res, next) => {
 };
 
 const dateValidation = Joi.date().required();
-
-const repetitoinSchema = Joi.object({
-  isRepeat: Joi.boolean().required(),
-  type: Joi.string().default("EVERY_DAY").required(),
-  week: Joi.number().min(1).max(3).required(),
-}).required();
-
+const repetitionSchema = Joi.object()
+  .keys({
+    isRepeat: Joi.boolean().required(),
+    type: Joi.string().default("EVERY_DAY").required(),
+    week: Joi.number().min(1).max(3).required(),
+  })
+  .required();
 const todoMemoSchema = Joi.string().max(200).required();
+const emailSchema = Joi.string()
+  .email({ minDomainSegments: 2, tlds: { allow: ["com", "net"] } })
+  .required();
 
-exports.verifyParams = verifyParams;
-
-exports.verifyDateRepetitoin = async (req, res, next) => {
+exports.verifyDaterepetition = async (req, res, next) => {
```

Comment:
```suggestion
exports.verifyDateRepetition = async (req, res, next) => {
```

아까 같이 보고 수정한 부분인데, 일괄변경하면서 카멜케이스에 어긋나게 됐네요. 번거로우시겠지만 수정해주시면 감사하겠습니다!